tests/diagnostic/test_diagnostics.h

```diff
@@ -35,9 +36,73 @@ std::array<std::uint32_t, GridLayout::dimension> fieldIndices(FieldFilter ff, Fu
         return {get(core::Direction::X), get(core::Direction::Y), get(core::Direction::Z)};
 }
 
+
+template<std::size_t dim, typename T, typename GridLayout, typename Field>
+void validateFluidGhosts(std::vector<T> const& data, GridLayout const& layout, Field const& field)
+{
+    using Filter = FieldDomainPlusNFilter;
```

Comment:
naive question : could that be now done with a MaskedView?

Reply:
possibly

Reply:
can we consider this a separate issue?